node/src/components/consensus.rs

```diff
@@ -201,6 +221,23 @@ where
                 sender,
                 proto_block,
             } => handling_es.handle_invalid_proto_block(era_id, sender, proto_block),
+            Event::GetValidatorsResponse {
+                switch_block_header,
+                get_validators_result,
+            } => match get_validators_result {
+                Ok(Some(result)) => {
+                    handling_es.handle_validators_response(*switch_block_header, result)
```

Comment:
nitpick: `handle_validators_response` sounds as if this method was handling a response _from_ validators.